src/styles/general/_variables.scss

```diff
@@ -1,14 +1,18 @@
 // Colors
 $color-black: #000;
 $color-white: #fff;
-$color-blue: #007bd3;
+$color-light-purple: #a78bfa;
+$color-medium-purple: #7c3aed;
+$color-dark-purple: #4c1d95;
 
 // Fonts
 $font-family: 'Roboto', sans-serif;
 
+// Font size
 $fs-small: 0.75rem; // 12px
 $fs-base: 16px;
 $fs-medium: 1.25rem; // 20px
+$fs-large: 30px;
```

Comment:
Everything is set with `rem` why you decided to use `px` here? Base is only thing allowed to be different